media-sound/frescobaldi/patches/frescobaldi-3.2.patchset

```diff
@@ -0,0 +1,89 @@
+From 3e2ea6a5e61f2ba50e672c2a63ace7f46367dced Mon Sep 17 00:00:00 2001
+From: =?UTF-8?q?Joachim=20Mairb=C3=B6ck?= <j.mairboeck@gmail.com>
+Date: Sat, 14 May 2022 18:05:48 +0200
+Subject: Haiku additions
+
+
+diff --git a/frescobaldi_app/hyphendialog.py b/frescobaldi_app/hyphendialog.py
+index 58119a1..443981c 100644
+--- a/frescobaldi_app/hyphendialog.py
++++ b/frescobaldi_app/hyphendialog.py
+@@ -56,6 +56,7 @@ default_paths = [
+     "share/scribus/dicts",
+     "share/scribus-ng/dicts",
+     "share/hunspell",
++    "data/hunspell",
+ ]
+ 
+ 
+@@ -68,7 +69,7 @@ def settings():
+ def directories():
+     """ Yields a list of existing paths based on config """
+     # prefixes to look in for relative paths
+-    prefixes = ['/usr/', '/usr/local/']
++    prefixes = ['/usr/', '/usr/local/', '/boot/system/', '/boot/home/config/']
```

Comment:
should the non-packaged variants also be added here?

Reply:
Yes, and /usr ones removed. Same below I think.

Reply:
I just added the Haiku specific paths for potentially upstreaming the changes (I have not tried that yet ...). But I think for the patchset you are right, Haiku needs only the paths relevant for it.

Reply:
There are a few more places where I added the Haiku specific paths instead of replacing the existing ones. Should these be changed too?